packages/core-components/src/components/product-list/FilterMobile.tsx

```diff
@@ -0,0 +1,246 @@
+        <div className='flex divide-x'>
+          <button className='px-3' onClick={changePageLayout}>
+            <svg
+              width='15'
+              height='16'
+              viewBox='0 0 15 16'
+              fill='none'
+              xmlns='http://www.w3.org/2000/svg'
+            >
+              <path
+                fillRule='evenodd'
+                clipRule='evenodd'
+                d='M0 0.5V7.16667H6.66667V0.5H0ZM5 5.5H1.66667V2.16667H5V5.5ZM0 8.83333V15.5H6.66667V8.83333H0ZM5 13.8333H1.66667V10.5H5V13.8333ZM8.33333 0.5V7.16667H15V0.5H8.33333ZM13.3333 5.5H10V2.16667H13.3333V5.5ZM8.33333 8.83333V15.5H15V8.83333H8.33333ZM13.3333 13.8333H10V10.5H13.3333V13.8333Z'
+                fill='#575757'
+              />
+            </svg>
+          </button>
+          <button
+            className='flex items-center px-3'
+            onClick={() => {
+              setIsOpenFilter(true);
+              // setIsOpenSort(false);
+            }}
+          >
+            <p className='pr-5'>{filterTitle}</p>
+            <svg
+              width='15'
+              height='16'
+              viewBox='0 0 15 16'
+              fill='none'
+              xmlns='http://www.w3.org/2000/svg'
+            >
+              <path
+                d='M0.23436 2.05625C0.328103 2.17812 5.61525 8.92812 5.61525 8.92812V14.5625C5.61525 15.0781 6.0371 15.5 6.56207 15.5H8.44632C8.96191 15.5 9.39313 15.0781 9.39313 14.5625V8.91875C9.39313 8.91875 14.5397 2.3375 14.7834 2.0375C15.0271 1.7375 14.999 1.4375 14.999 1.4375C14.999 0.921875 14.5772 0.5 14.0522 0.5H0.946813C0.374975 0.5 0 0.95 0 1.4375C0 1.625 0.0562463 1.85 0.23436 2.05625Z'
+                fill='#575757'
+              />
+            </svg>
+          </button>
+        </div>
+      </div>
+      {/* List Sort */}
+      {filterMobileItems?.sortItems && isOpenSort ? (
+        <div className='relative'>
+          <ul className='bg-white divide-y text-sm font-normal w-full z-0 absolute opacity-100 top-0 transition-all ease-in duration-500'>
+            {filterMobileItems.sortItems.filterOptions.map((item) => {
+              return item.selected ? (
+                <li key={item.optionId} className='px-3 py-5 text-blue-600'>
+                  <p>{item.optionName}</p>
+                </li>
+              ) : (
+                <li
+                  key={item.optionId}
+                  className='px-3 py-5 hover:bg-gray-200 cursor-pointer'
+                  onClick={() => {
+                    onChangeSort([
+                      formatFilterValue(
+                        filterMobileItems.sortItems.filterType,
+                        item.optionId
+                      ),
+                    ]);
+                  }}
+                >
+                  <p>{item.optionName}</p>
+                </li>
+              );
+            })}
+          </ul>
+        </div>
+      ) : (
+        <></>
+      )}
+      {/* List Filter */}
+      {isOpenFilter ? (
```

Comment:
too many render condition here, a? reason why we need to split into smaller components